WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -329,7 +329,16 @@ extension OrdersViewController {
         }
 
         ServiceLocator.analytics.track(.ordersListSearchTapped)
-        let searchViewController = OrderSearchViewController(storeID: storeID)
+
+        let storageManager = ServiceLocator.storageManager
+        let descriptor = NSSortDescriptor(keyPath: \StorageOrder.dateCreated, ascending: false)
+
+        let searchResultsController = ResultsController<StorageOrder>(storageManager: storageManager, sortedBy: [descriptor])
```

Comment:
I would suggest encapsulating the creation of the results controller in the SearchViewController itself, it seem to me like the fact that searches are performed with the help of a results controller should be an implementation detail and should not be leaked.

Reply:
Or it could be provided by the view model (what is now called the SearchCommand)

Reply:
good point, I moved the results controller in https://github.com/woocommerce/woocommerce-ios/pull/1306/commits/a8f0ac35b42f9a1d53edce00fae3fadfc6ca816a